web/repositories.bzl

```diff
@@ -13,84 +13,74 @@
-        build_file=prefix + "//build_files:BUILD.selenium_java",
-        sha256="a26a449388abd46d1e152771e3641859ac4acee9c0ea24a101ca369048a81ecb",
-        url="http://selenium-release.storage.googleapis.com/3.0-beta3/selenium-java-3.0.0-beta3.zip",
-    )
+    if not omit_org_seleniumhq_java:
+      native.new_http_archive(
+          name="org_seleniumhq_java",
+          build_file=str(Label("//build_files:BUILD.selenium_java")),
+          sha256="a26a449388abd46d1e152771e3641859ac4acee9c0ea24a101ca369048a81ecb",
+          url="http://selenium-release.storage.googleapis.com/3.0-beta3/selenium-java-3.0.0-beta3.zip",
+      )
 
-    native.maven_jar(
-        name="org_json_json",
-        artifact="org.json:json:20160810",
-        sha1="aca5eb39e2a12fddd6c472b240afe9ebea3a6733",)
+    if not omit_org_json_json:
+      native.maven_jar(
+          name="org_json_json",
+          artifact="org.json:json:20160810",
+          sha1="aca5eb39e2a12fddd6c472b240afe9ebea3a6733",)
 
-    native.maven_jar(
-        name="com_google_code_findbugs_jsr305",
-        artifact="com.google.code.findbugs:jsr305:3.0.1",
-        sha1="f7be08ec23c21485b9b5a1cf1654c2ec8c58168d",)
+    if not omit_com_google_code_findbugs_jsr305:
+      native.maven_jar(
+          name="com_google_code_findbugs_jsr305",
+          artifact="com.google.code.findbugs:jsr305:3.0.1",
+          sha1="f7be08ec23c21485b9b5a1cf1654c2ec8c58168d",)
 
-    native.maven_jar(
-        name="com_google_guava_guava",
-        artifact="com.google.guava:guava:19.0",
-        sha1="6ce200f6b23222af3d8abb6b6459e6c44f4bb0e9",)
+    if not omit_com_google_guava_guava:
+      native.maven_jar(
+          name="com_google_guava_guava",
+          artifact="com.google.guava:guava:19.0",
+          sha1="6ce200f6b23222af3d8abb6b6459e6c44f4bb0e9",)
 
   if go:
-    native.new_git_repository(
-        name="com_github_tebeka_selenium",
-        build_file=prefix + "//build_files:BUILD.selenium_go",
-        remote="https://github.com/tebeka/selenium.git",
-        tag="v0.8.5",)
+    if not omit_com_github_tebeka_selenium:
+      native.new_http_archive(
+          name="com_github_tebeka_selenium",
+          build_file=str(Label("//build_files:BUILD.selenium_go")),
+          url="https://github.com/tebeka/selenium/archive/v0.9.2.tar.gz",
+          sha256="c5f21652eda6230ee8bb5f9f02b740fa8d8b22c0cddc832ec666a7654bb0d1a4",
+          strip_prefix="selenium-0.9.2",)
 
   if python:
-    native.new_http_archive(
-        name="org_seleniumhq_py",
-        build_file=prefix + "//build_files:BUILD.selenium_py",
-        sha256="0705803349964c7a2a144f1796a5d29905fe2a09931b2bb945ee0cb4deab75d7",
-        strip_prefix="selenium-3.0.1/py",
-        url="https://pypi.python.org/packages/3a/a3/e4ab60a0229a85f468a36367bc0672a4bca2720f24391eda33704a5f0ad5/selenium-3.0.1.tar.gz",
-    )
+    if not omit_org_seleniumhq_py:
+      native.new_http_archive(
+          name="org_seleniumhq_py",
```

Comment:
[Optional, possibly better for future PR] I would recommend the name org_python_pypi_selenium. It's not org_selenium because it's not the canonical source code tarball distributed by seleniumhq.org. It's just a Python release tarball uploaded specifically for PyPi.

Speaking of which, Closure Rules probably needs to be more rigorous in these names. But I've been waiting for @kchodorow to lead the way on enforcing a naming convention.